.pre-commit-config.yaml

```diff
@@ -7,17 +7,18 @@ repos:
         args: [--safe, --quiet]
         language_version: python3
```

Comment:
Didn't we have a pre-commit hook for the `encoding` comments? 🤔

Reply:
yep, and now it is enforcing that they are removed `fix-encoding-pragma` w/ `args: [--remove]`